test/factory.spec.ts

```diff
@@ -649,19 +649,6 @@ describe('Factory', () => {
       });
     });
 
-    it('extends the factory with new attributes', async () => {
```

Comment:
Removed because this is not a valid usage anymore. We shouldn't be able to define properties in the factory that are not present in the model